### PR TITLE
Transfer styles to the App.css file

### PR DESCRIPTION
**Pull request by adam-lzk**
Transfer styles to the App.css file, but they don't work correctly

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,131 +1,49 @@
...
 function App() {
   return (
   <Space
     direction="vertical"
-    style={{
-      width: '100%',
-      margin: 0
-
-    }}
+    className='space'
```
Comment (dmlaziuk) [inline src/App.jsx]: Please delete `Space` component.
`Layout` should be the root component.

File: src/App.jsx [hunk 2]
```diff
@@ -1,131 +1,49 @@
...
           для маленьких гостей,  наших генацваликов!</p>
-          <p style={pStyle}>Дорогие наши, приглашаем вас в семью ресторанов Хинкальня в вашем городе — Минске, Бресте,
+          <p>Дорогие наши, приглашаем вас в семью ресторанов Хинкальня в вашем городе — Минске, Бресте,
           Гродно, Мозыре, Солигорске и Пинске! Выбирайте в нашем меню свои любимые блюда, а мы их с
           любовью приготовим, хорошенько упакуем и привезем. Горячими, сочными, ароматными!</p>
         </Content>
-        <Footer style={footerStyle}>
-          <div style={citiesStyle}>
+        <Footer className='footer'>
+          <div className='cities'>
```
Comment (dmlaziuk) [inline src/App.jsx]: Please remove cities for now.

File: src/App.jsx [hunk 3]
```diff
@@ -1,131 +1,49 @@
...
   >
     <Layout>
-      <Sider style={siderStyle}>
-        <button style={buttonStyle}>Главная страница</button>
-        <button style={buttonStyle}>Рестораны</button>
-        <button style={buttonStyle}>Заказать доставку</button>
-        <button style={buttonStyle}>Вакансии</button>
-        <button style={buttonStyle}>Оплата</button>
-        <button style={buttonStyle}>Контакты</button>
+      <Sider className='sider'>
```
Comment (dmlaziuk) [inline src/App.jsx]: Let's implement the following layout:
```
Fixed header
-------------------------------
Fixed      | Scrollable content
side       | with footer
navigation | 
```

https://github.com/adam-lzk/restaurant-website-with-Ant-Design/assets/32341162/a0dd5f0e-5de8-450f-865b-053d20558458

File: src/App.jsx [hunk 4]
```diff
@@ -1,131 +1,49 @@
...
+        <button>Главная страница</button>
+        <button>Рестораны</button>
+        <button>Заказать доставку</button>
+        <button>Вакансии</button>
+        <button>Оплата</button>
+        <button>Контакты</button>
```
Comment (dmlaziuk) [inline src/App.jsx]: Please use `Menu` component from `antd`
```suggestion
          <Menu
            items={[
              { key: "1", label: "Label 1" },
              { key: "2", label: "Label 2" },
              { key: "3", label: "Label 3" },
              { key: "4", label: "Label 4" },
              { key: "5", label: "Label 5" },
              { key: "6", label: "Label 6" },
            ]}
          />
```

File: src/App.jsx [hunk 5]
```diff
@@ -1,131 +1,49 @@
...
-          <h2 style={h2Style}>О нас</h2>
-          <p style={pStyle}>Хинкальня — это семья, здесь берегут рецепты любимых блюд и готовят только самое вкусное:
+        <Header className='header'>Хинкальня</Header>
+        <Content className='content'>
+          <img
+            src="../pictures/pict1.jpg"
+            alt="Хинкали"
+          />
+          <h2>О нас</h2>
+          <p>Хинкальня — это семья, здесь берегут рецепты любимых блюд и готовят только самое вкусное:
```
Comment (dmlaziuk) [inline src/App.jsx]: Please don't use russian language

File: src/App.css [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  text-align: center;
+  padding: 10;
+  padding-right: 20;
+  padding-left: 20;
+  margin: 20;
+  border: 0;
+  border-radius: 25px;
+  color: #fff;
+  background-color: #af2127;
+}
```
Comment (dmlaziuk) [inline src/App.css]: We'll be using antd design system for UI components.
It has the proper margins, paddings and colors. 
```suggestion
```

File: src/App.css [hunk 7]
```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+}
+
+.space {
+  width: 100%;
+  margin: 0;
+}
```
Comment (dmlaziuk) [inline src/App.css]: There'll be no `Space`
```suggestion
```

File: src/App.css [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+.header {
+  font-family: Georgia;
+  text-align: center;
+  font-size: 30px;
+  color: #fff;
+  height: 120;
+  line-height: 120px;
+  padding-inline: 50;
+  background-color: #af2127;
```
Comment (dmlaziuk) [inline src/App.css]:
```suggestion
  position: sticky;
  ... from https://ant.design/components/layout#components-layout-demo-fixed
```

File: src/App.css [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.content {
+  font-family: sans-serif;
+  font-size: 16;
+  min-height: 120;
+  line-height: 35px;
+  color: #260000;
+  background-color: #fcf5f5;
+  padding-bottom: 5;
```
Comment (dmlaziuk) [inline src/App.css]:
```suggestion
  overflow: auto; // for scrolling
```

File: src/App.css [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.sider {
+  text-align: center;
+  line-height: 90px;
+  color: #000;
+  background-color: #fff0f0;
+  padding-top: 20;
```
Comment (dmlaziuk) [inline src/App.css]:
```suggestion
  height: calc(viewport height minus header height);
```

File: src/App.css [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.footer {
+  text-align: center;
+  color: #fff;
+  background-color: #af2127;
```
Comment (dmlaziuk) [inline src/App.css]: Let's keep footer as is for now.
```suggestion
```

File: src/App.css [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+.cities {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 60px;
+  padding-right: 225;
+  padding-left: 225;
+  align-items: center;
+  font-size: 16;
+}
```
Comment (dmlaziuk) [inline src/App.css]: Let's remove it for now
```suggestion
```

**Review by dmlaziuk — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

File: src/App.jsx (hunk 2 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

File: src/App.jsx (hunk 3 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

File: src/App.jsx (hunk 4 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

File: src/App.jsx (hunk 5 above)
Comment (adam-lzk) [reply src/App.jsx]: Fixed

File: src/App.css (hunk 6 above)
Comment (adam-lzk) [reply src/App.css]: Fixed

File: src/App.css (hunk 7 above)
Comment (adam-lzk) [reply src/App.css]: Fixed

File: src/App.css (hunk 8 above)
Comment (adam-lzk) [reply src/App.css]: Fixed

File: src/App.css (hunk 9 above)
Comment (adam-lzk) [reply src/App.css]: Fixed

File: src/App.css (hunk 10 above)
Comment (adam-lzk) [reply src/App.css]: Fixed

File: src/App.css (hunk 11 above)
Comment (adam-lzk) [reply src/App.css]: Fixed

File: src/App.css (hunk 12 above)
Comment (adam-lzk) [reply src/App.css]: Fixed

File: src/App.jsx (hunk 3 above)
Comment (dmlaziuk) [reply src/App.jsx]: 👍

**Review by dmlaziuk — APPROVED** [review]
Great work 👍

**Pull request merged**